code/datums/mutantraces.dm

```diff
@@ -1111,6 +1111,10 @@ TYPEINFO(/datum/mutantrace)
 		if(ishuman(M))
 			M.mob_flags |= IS_BONEY
 			M.blood_id = "calcium"
+			M.head_tracker = M.organHolder.head
+			head = M.organHolder.head
+			head.is_skeleton = TRUE
```

Comment:
Just realized that `head_type` is a thing, which already tracks `HEAD_SKELETON`.

Reply:
Used head_type instead of is_skeleton from now on.